### ISSUE TITLE
Building instructions... insufficient

### ISSUE THREAD

**Issue by relascope** [unknown-horizons/unknown-horizons#2323]
since you are also helping in the development of FIFE and there is no fast option to post,
here is my experience in building UH so far...

everything compiled/installed without errors. 
[except that I did 
sudo make install
instead of
make install
for fifechan]

when I try to run ./run_uh.py
Failed to use FIFE from <module 'fife' from '/home/dojoy/src/games/fifengine/engine/python/fife/__init__.pyc'>
libfifechan.so.0.1.2: cannot open shared object file: No such file or directory
Failed to find and/or load FIFE.

libfifechan.so.0.1.2 is in /usr/local/lib/

What's next?

**Comment by stubb** [comment]
If you are using an own compiled version of fife, you need to recompile it.

**Comment by stubb** [comment]
if you installed it via the package manager apt-get you did not compiled it from source yourself ;-)
So http://www.unknown-horizons.org/download/linux/ are the correct instructions, and yes that is the preferred way.

About the libfifechan issue, please start UH with the debug flag -d and paste the output, also take a look at the fife build folder and paste the build.log file please

**Comment by relascope** [comment]
once again...
Post 1)
I tried to compile from source, since I'm a developer myself and so I can find build-issues...

for Post 2) 
I tried to find the fastest way to get it running
and that was 
apt-get install python-fife

now for your post (the debug...)

trying to recreate using a new Debian Jessie VM...
./setup.py build_i18n
needs intltool (should be in apt-get list...)
game "runs"... no X-Server so not really, but the error not finding libfifechan... is gone...

starting over again I discovered, that I didn't call setup.py the first time (then the error is here again...). 
As I remember, I didn't really find the building instructions for UH and tried on my own, so I didn't know how to call setup. Suggesting a short message about how to run ./setup.py in a readme/install.. in the build directory to be independent of HP, when already having source....

Anyway, sorry for bothering you!
